contracts/EventFactory.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.8.0;
+import "./Event.sol"; //import Event contract
+import "@optionality.io/clone-factory/contracts/CloneFactory.sol"; //This only works after you run: npm install @optionality.io/clone-factory
```

Comment:
In my PR, I created a package.json file to track dependencies. Can you do an npm install --save-dev on this package to add it that eventually.